src/test/java/com/fasterxml/jackson/databind/deser/jdk/ObjectArrayDeserializer5165Test.java

```diff
@@ -0,0 +1,52 @@
+package com.fasterxml.jackson.databind.deser.jdk;
+
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.exc.InvalidNullException;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+// For [databind#5165]
+public class ObjectArrayDeserializer5165Test
```

Comment:
These tests are not valid: no `null`s are being read at all!
I'll see if I can fix them.

Reply:
... Ahhhhhhh. No, even more clever, relies on "" being coerced (by default) into `null`s, for `Integer` values.